doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function does not work on AIX versions older than 7.1. It can still
+be called but will return `UV_ENOSYS`.
+
```

Comment:
I like the idea of linking the two sentences, but that suggests that `UV_ENOSYS` is a reasonable thing for it to return. How about:

```js
*Note*: This function does not work on AIX versions older than 7.1, 
it will return the error `UV_ENOSYS`.
```